examples/onnxruntime/training/docker/Dockerfile-cu11-ort11

```diff
@@ -0,0 +1,37 @@
+# Use nvidia/cuda image
+FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04 as builder
```

Comment:
Do we need `as builder` ? It seems you're not reusing the `builder` context afterwards

Reply:
Yeah, I am not using it after. I will remove it thanks for the comment!